projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DeployJob.tsx

```diff
@@ -84,19 +85,39 @@ export async function showCreateDeploymentDialog(statusButton?: StatusButton) {
             await jobRequest('deploy');
           }
         } else {
-          await showErrorMessage(
-            'Encоuntered an error while running the job!',
-            message,
-            [Dialog.okButton()]
-          );
+          await showDialog({
+            title: CREATE_DEP_BUTTON_LABEL,
+            body: (
+              <div className="vdk-error-dialog">
```

Comment:
Be careful of magic strings. If you are using a string in multiple places, consider putting it as a constant

Reply:
Agreed. Will create a constant about it.

Reply:
Done.